examples/crm/src/dashboard/TasksList.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { Card, Box, Button } from '@mui/material';
+import AssignmentTurnedInIcon from '@mui/icons-material/AssignmentTurnedIn';
+import {
+    useGetList,
+    Link,
+    useGetIdentity,
+    useList,
+    ListContextProvider,
+    ResourceContextProvider,
+} from 'react-admin';
+import { TasksIterator } from '../tasks/TasksIterator';
+
+import { Contact } from '../types';
+
+export const TasksList = () => {
+    const { identity } = useGetIdentity();
+
+    // get all the contacts for this sales
+    const { data: contacts, isPending: contactsLoading } = useGetList<Contact>(
+        'contacts',
+        {
+            pagination: { page: 1, perPage: 500 },
```

Comment:
Yes, that's a lot of contacts, but since we can't do a join there is no lighter way of fetching the tasks of the contacts followed by the current sales.